### PR TITLE
Fix a typo in 2.1.17 release notes

### PR DESCRIPTION
**Pull request by XiaoJenny**
### What is changed, added or deleted? (Required)

tomestone -> tombstone: to close https://github.com/pingcap/docs-cn/issues/2669

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: releases/release-2.1.17.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ TiDB Ansible 版本：2.1.17
 - 新特性
     - TiDB 的 `SHOW TABLE REGIONS` 语法新增 `WHERE` 条件子句
     - TiKV、PD 新增 `config-check` 功能，用于配置项检查
-    - pd-ctl 新增 `remove-tomestone` 命令，支持清理 tombstone store 记录
+    - pd-ctl 新增 `remove-tomstone` 命令，支持清理 tombstone store 记录
```
Comment (lilin90) [inline releases/release-2.1.17.md:17]:
```suggestion
    - pd-ctl 新增 `remove-tombstone` 命令，支持清理 tombstone store 记录
```

Comment (ran-huang) [inline releases/release-2.1.17.md:17]:
```suggestion
    - pd-ctl 新增 `remove-tombstone` 命令，支持清理 tombstone store 记录
```

**Comment by ran-huang** [comment]
Thank you for your contribution! 😁
I just made a small edit.

**Comment by XiaoJenny** [comment]
@lilin90 PTAL

**Pull request merged**